src/prefect/logging/handlers.py

```diff
@@ -118,7 +118,7 @@ def flush(cls) -> None:
 
             # Not ideal, but this method is called by the stdlib and cannot return a
             # coroutine so we just schedule the drain in the global loop thread and continue
```

Comment:
comment should probably be updated to match new behavior
```suggestion
            # coroutine so we just schedule the drain in a new thread and continue
```